### DISCUSSION TITLE
Why base10 file sizes?

### DISCUSSION THREAD

**Discussion by informatdcs** [PlexRipper/PlexRipper#305] [General]
I was going trough all the media I had access to and then I donwloaded one file just to see that it didnt match with the size displayed on the webpage, I've read that pretty-bytes uses base10 for conversion which in my opinion is more clear (all code inside pretty-bytes but displays all size data not according to the real file size the user will perceive, we are talkin 5%~ of difference but still, it is different.

![image](https://github.com/PlexRipper/PlexRipper/assets/5058480/97fd51cf-a24a-4171-8b15-1b9882a9de88)
![image](https://github.com/PlexRipper/PlexRipper/assets/5058480/fb2243ad-337a-43c1-9d18-40e8b64a445e)

Could it be possible to revert to the last way it was done? I've seen it and it was posible to use 1024 in the calc

**Comment by JasonLandbridge** [comment]
Forgot about this question, this was indeed a problem but has been fixed in the meantime. Sizes should correspond with how Plex displays them
